sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -465,5 +465,8 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
 /**
  * The default time to wait for messages _after_ the first message
  * has been received.
+ * 
+ * @internal
+ * @ignore
  */
-const maxTimeAfterFirstMessageMs = 1000;
+export const defaultMaxTimeAfterFirstMessageMs = 1000;
```

Comment:
Nit:

This applies only to the `receiveMessages()` call and not the iterator or the subscribe() method.
Perhaps, consider renaming it and update docs to reflect the same?
Else one might consider that this applies to all types of receiving

Reply:
Yeah, that makes sense.

Reply:
Done. Gave it a little polish :)